be/src/olap/olap_engine.cpp

```diff
@@ -1725,14 +1725,18 @@ void OLAPEngine::get_tablet_stat(TTabletStatResult& result) {
     int64_t current_time = UnixMillis();
     
     _tablet_map_lock.rdlock();
```

Comment:
_tablet_map_lock should not be locked here.
It should be locked inside _build_tablet_stat()

Reply:
OK.